setup.py

```diff
@@ -64,7 +64,8 @@
     'Topic :: Security',
     'Topic :: Software Development'
   ],
-  packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
+  packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests",
+      "debian"]),
   install_requires=["six", "securesystemslib==0.10.8", "attrs",
                     "python-dateutil", "iso8601"],
```

Comment:
Unrelated:
Looking at `debian/control`, where you correctly added cryptography as a dependency (re-introduced in #150), I realized that we have to re-add it here and in `requirements.txt` as well. Worth an extra PR?

Reply:
Fixed with #174

Reply:
kk, now I get what you mean. I'll rebase in a sec.